iceoryx_hoofs/test/moduletests/test_posix_signal_watcher.cpp

```diff
@@ -85,21 +85,21 @@ void unblocksWhenSignalWasRaisedForWaiters(SignalWatcher_test& test,
                                            const uint64_t numberOfWaiters,
                                            const std::function<void()>& wait)
 {
-    std::atomic<uint64_t> isThreadStarted{0};
+    Barrier isThreadStarted(numberOfWaiters);
```

Comment:
Hahaha, I didn't know this exists.

Not required but it feels a bit weird to read `isThreadStarted.wait()`. Can you come up with a more lyrical name like `threadStartSync.wait()` of so?

We can keep it if you like it as it is or don't find something that satisfies you.